packages/admin/src/pages/build-application/[applicationId]/[sectionId]/[questionId]/delete-confirmation.test.tsx

```diff
@@ -79,32 +79,17 @@ describe('Delete question page', () => {
       expect(document.title).toBe('Error: Delete a question - Manage a grant');
     });
 
-    it('Should render a back button with correct link on it', () => {
-      render(component);
-      expect(screen.getByRole('link', { name: 'Back' })).toHaveAttribute(
-        'href',
-        '/dashboard'
-      );
-    });
-
     it('Renders page layout', () => {
       render(component);
       screen.getByTestId('question-page-form');
-    });
-
-    it('Renders radio input to confirm whether to delete question or not', () => {
-      render(component);
       screen.getByTestId('radioFormDiv');
-    });
-
-    it('Renders "Confirm" button', () => {
-      render(component);
       screen.getByRole('button', { name: 'Confirm' });
-    });
-
-    it('Renders "Cancel" link', () => {
-      render(component);
       screen.getByRole('link', { name: 'Cancel' });
+      expect(document.title).toBe('Delete a question - Manage a grant');
+      expect(screen.getByRole('link', { name: 'Back' })).toHaveAttribute(
+        'href',
+        '/dashboard'
+      );
     });
```

Comment:
I'm not really too fussed about this but usually I prefer separated unit tests, easier to identify any failures and not prone to fixing one failure, to only get lower down for it to fail again etc

Reply:
generally i would agree but in this case the tests are a bit too simple and similar to be in separate tests (they're literally all just checking that some content rendered on the component with default props)